pylab/website/tests/test_comments.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+
+from django_webtest import WebTest
+
+from django.contrib.auth.models import User
+
+from django_comments.models import Comment
+
+from pylab.core.models import Project
+
+
+class CommentsTests(WebTest):
+
+    def setUp(self):
+        self.user = User.objects.create_user('u1')
+        self.project = Project.objects.create(
+            author=self.user,
+            title='Test project',
+            description='Description',
+            created='2015-08-13'
+        )
+
+    def test_add_comment(self):
+        resp = self.app.get('/projects/test-project/', user=self.user)
+        now = datetime.datetime.now()
+        comment = Comment.objects.create(
```

Comment:
If I fill the form like this
```
resp.form['comment'] = 'new comment'
resp = resp.form.submit()
```
it is not saved in the database. 
Maybe there is an issue considering that I use django_comments and not self created form?